### ISSUE TITLE
Static paths with "+" signs in them won't load.

### ISSUE THREAD

**Issue by miladmostavi** [laravel/valet#320]
My driver's isStaticFile is the standard one:

```
public function isStaticFile($sitePath, $siteName, $uri)
{
    if (file_exists($staticFilePath = $sitePath.'/app/public_html/'.$uri)) {
        return $staticFilePath;
    }

    return false;
}
```

All files works fine, except: `/js/vendor/history.js/scripts/bundled/html4+html5/jquery.history.js`

**Comment by miladmostavi** [comment]
I temporarily solved my issue by replacing file names which have spaces in them with `+`.

```
public function isStaticFile($sitePath, $siteName, $uri)
{
    $staticFilePath = $sitePath.'/app/public_html'.$uri;
    if(preg_match('/\s/', $staticFilePath) && ! file_exists($staticFilePath)) {
        $staticFilePath = str_replace(' ', '+', $staticFilePath);
    }

    if (file_exists($staticFilePath)) {
        return $staticFilePath;
    }

    return false;
}
```

The `$uri` that is received by the function comes with a space instead of the original `+` sign.

**Comment by mattstauffer** [comment]
Closed via #830
